src/main/scala/com/lightbend/rp/sbtreactiveapp/magic/Lagom.scala

```diff
@@ -36,9 +36,12 @@ object Lagom {
     }
   }
 
-  def endpoints(classPath: Seq[Attributed[File]], scalaLoader: ClassLoader): Option[Map[String, Endpoint]] =
+  def endpoints(classPath: Seq[Attributed[File]], scalaLoader: ClassLoader): Option[Seq[Endpoint]] =
     services(classPath, scalaLoader).map(decodeServices)
 
+  def hasCluster(allDependencies: Seq[ModuleID]): Boolean =
+    allDependencies.exists(l => l.organization == "com.lightbend.lagom" && l.name.contains("-persistence-"))
```

Comment:
:face_with_head_bandage:

Reply:
I think we need to detect more than `-persistence-` - don't we also need to detect all the modules that Lagom has that requires clustering, i.e. the cluster module itself for example?